### ISSUE TITLE
jx create cluster command - error creating cluster exit status 1 

### ISSUE THREAD

**Issue by jhendrick** [jenkins-x/jx#1002]
I'm trying to setup Jenkins X on my MacBook Pro (High Sierra 10.13.4).  I've downloaded the latest version of Jenkins X (1.2.117) and tried to create a cluster on GKE.  It fails as seen in the screenshot below.  How can I get more details on how to debug this?

<img width="1497" alt="screen shot 2018-06-09 at 2 29 43 pm" src="https://user-images.githubusercontent.com/12519171/41196310-9889a636-6bf1-11e8-94aa-ebcd470bf2f7.png">

Joshuas-MBP:Downloads jhendrick$ jx create cluster gke
? Google Cloud Project: bubbly-tractor-175801
No cluster name provided so using a generated one: handglacier
? Google Cloud Zone: us-west1-a
? Google Cloud Machine Type: n1-standard-4
? Minimum number of Nodes 3
? Maximum number of Nodes 5
Creating cluster...
Error: Command failed  gcloud container clusters create handglacier --zone us-west1-a --num-nodes 3 --machine-type n1-standard-4 --enable-autoscaling --min-nodes 3 --max-nodes 5 --labels=created-by=jhendrick
error creating cluster exit status 1
error: exit status 1

**Comment by rasmusdencker** [comment]
Just to clarify this issue was sorted out on the slack channel. The problem became obvious when running the `gcloud container clusters create` command by itself; a GCP quota limit was reached. 

Might be worth routing stderr from the subcommands to the shell in case a command fails to make errors more transparent and debuggable

**Comment by jhendrick** [comment]
Closing the issue.  A bit more detail.  I ran:

jhendrick$ gcloud container clusters create carpetrattle --zone us-west1-a --num-nodes 3 --machine-type n1-standard-4 --enable-autoscaling --min-nodes 3 --max-nodes 5 --labels=created-by=jhendrick
WARNING: Currently node auto repairs are disabled by default. In the future this will change and they will be enabled by default. Use `--[no-]enable-autorepair` flag  to suppress this warning.
WARNING: Starting in Kubernetes v1.10, new clusters will no longer get compute-rw and storage-ro scopes added to what is specified in --scopes (though the latter will remain included in the default --scopes). To use these scopes, add them explicitly to --scopes. To use the new behavior, set container/new_scopes_behavior property (gcloud config set container/new_scopes_behavior true).
ERROR: (gcloud.container.clusters.create) ResponseError: code=403, message=Insufficient regional quota to satisfy request for resource: "CPUS". The request requires '12.0' and is short '4.0'. The regional quota is '8.0' with '8.0' available.

Reduced to n1-standard-1 with 1 to 5 nodes and all worked.
